apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/annotations/_components/RunsPage.tsx

```diff
@@ -99,6 +103,18 @@ export function RunsPage({
       },
       { keepPreviousData: true },
     )
+  const hitlAnnotations = useMemo(() => {
+    return annotations.filter((a) => {
+      const evaluation = evaluations.find((ev) => ev.uuid === a.evaluationUuid)
+      if (!evaluation) return false
+
+      return (
+        evaluation.type === EvaluationType.Human &&
+        (evaluation as EvaluationV2<EvaluationType.Human>).configuration
```

Comment:
This is supper weird. Why we need to filter annotations. Annotations are HITL evaluations

Reply:
The question is why annotations are not already filtered